kubernetes/types.go

```diff
@@ -641,3 +641,10 @@ func (in *GenericIstioObjectList) DeepCopyObject() runtime.Object {
 	}
 	return nil
 }
+
+func (imc IstioMeshConfig) GetEnableAutoMtls() bool {
```

Comment:
I have a concern to grow business logic inside kubernetes/ package instead of business/ model/ ones.
Not a blocker, but at some point we need to revisit this logic to try to not have a mix of logic in the kubernetes/ and business + model ones.